govcd/edgegateway.go

```diff
@@ -634,3 +641,89 @@ func (eGW *EdgeGateway) RemoveIpsecVPN() (Task, error) {
 	}
 	return eGW.AddIpsecVPN(ipsecVPNConfig)
 }
+
+// Deletes the edge gateway, returning a task and an error if the vCD call fails.
```

Comment:
Is the comment really right? Isn't the function returning a task even when delete succeeds?

Reply:
fixed